outlook/src/taskpane/components/Company/CompanySection/CompanySection.tsx

```diff
@@ -106,50 +107,64 @@ class CompanySection extends React.Component<CompanySectionProps, CompanySection
 
     }
 
-    private getCompanyLayout= () => {
-        const company = this.state.company as Company;
-
-        if (company.isEmpty())
+    private updateCompanyRequest = () => {
+        if (!this.props.partner.email)
```

Comment:
For new or reworked code, I suggest we fall back to Odoo JS best practice.

Meaning 
```js
if (condition) {
    // code
}
```


Instead of 
```js
if (condition)
{
    // code
}
```

Reply:
That is correct, regarding the linter problem, it has been installed since the last PR which was before this PR, that's why there are some linting problems here and there, but these are being corrected now.